starter-dev-backend/src/schema/github/github.typedefs.ts

```diff
@@ -37,15 +37,28 @@ export const githubTypeDefs = gql`
     location: String
     login: String
     name: String
+    orgs: [Orgs]
+    starred_url: String
     twitter_username: String
   }
 
+  """
+  A Organization object used in Owner
```

Comment:
Suggestion: Grammar 😛

```suggestion
  An Organization object used in Owner
```